doc/Command_Index_User.tex

```diff
@@ -790,16 +790,23 @@ \subsubsection{End of Line Behavior}
 \subsubsection{Hyphenation}
 
 \macroname{\textbackslash gresethyphen}{\{\#1\}}{gregoriotex-main.tex}
-Tell Gregorio\TeX\ to when to place a hyphen between
-syllables in polysyllabic words in a score.  This is done by overriding \texttt{maximumspacewithoutdash} so subsequent changes to this dimension will override this command.
+Tells Gregorio\TeX\ how to place a hyphen between syllables in polysyllabic words in a score.  This is done by overriding \texttt{maximumspacewithoutdash} so subsequent changes to this dimension will override this command.
```

Comment:
Let's tackle that as a separate issue.

It would be a little more complex than what you mention.  I introduced that flag so I could have an "or" condition that causes the hyphen to appear: either it's the first initial and the hyphen is forced or the spacing requires a hyphen.  As such, this flag is reset by \GreSyllable.  We could use it for `\gresethyphen`, which should then need to set its own flag (rather than adjusting `maximumspacewithoutdash`) which would then be used as an additional clause of the above "or" condition.